controller/controller-common/src/main/java/com/pinterest/rocksplicator/controller/Task.java

```diff
@@ -27,10 +27,12 @@
  */
 public class Task extends TaskBase {
 
-  public static int PENDING = 0;
-  public static int RUNNING = 1;
-  public static int DONE = 2;
-  public static int FAILED = 3;
+  public enum TaskState {
```

Comment:
There is already a TaskState enum defined in controller-common. You can reuse that one.